hyperframe/frame.py

```diff
@@ -381,20 +391,21 @@ def __init__(self, stream_id=0, settings=None, **kwargs):
         self.settings = settings or {}
 
     def serialize_body(self):
-        settings = [struct.pack("!HL", setting & 0xFF, value)
-                    for setting, value in self.settings.items()]
-        return b''.join(settings)
+        return b''.join([_STRUCT_HL.pack(setting & 0xFF, value)
+                         for setting, value in self.settings.items()])
```

Comment:
Given that we're making performance optimisations and you want to squash this down to one line, let's save ourselves an intermediate allocation and remove the square brackets. `.join()` takes an iterable, so there's no need to create an intermediate list: a generator comprehension is just fine.